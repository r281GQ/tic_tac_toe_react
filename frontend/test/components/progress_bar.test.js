import React from 'react';
import { shallow } from 'enzyme';
import ProgressBar from './../../src/components/progress_bar';

describe('ProgressBar test', () => {
  it('should render', () => {
    const component = shallow(
      <ProgressBar/>
    );
    expect(component).toHaveLength(1);
  });
});
