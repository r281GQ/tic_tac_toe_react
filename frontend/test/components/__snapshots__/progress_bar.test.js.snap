// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar test should render 1`] = `
<div
  className="progress__absent"
/>
`;
